convert.h

```diff
@@ -38,6 +38,7 @@
 #include <netinet/in.h>
 
 #define CONVERT_VERSION 1
+#define CONVERT_MAGIC_NO 0x2263
```

Comment:
or 8803 in decimal format :)
(like the RFC)
But best to keep 0x2263 as specified on the RFC